docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+services:
+  db:
+    image: postgres:13.9
+    container_name: phpreport-db
+    ports:
+      - "5432:5432"
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: phpreport
+      POSTGRES_USER: phpreport
+      POSTGRES_DB: phpreport
+  phpreport-app:
+    build:
+      context: ../
+      dockerfile: docker/dev.app.Dockerfile
+    container_name: phpreport-app
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
```

Comment:
I think it's better to remove this container, we are not planning to deploy it to prod like this and if we need to do it than we'd also need to make a production dockerfile for it, not use the dev version that spawns the built-in php web server and it's not production ready.